editor/questionbankcontroller.js

```diff
@@ -520,6 +521,105 @@ angular.module('createquestionapp', [])
         ecEditor.dispatchEvent($scope.pluginIdObj.question_create_id + ":showpopup", questionObj);
       }
     }
+    $scope.copyQuestion = function(questionObj){
```

Comment:
Why have you repeated the save question code here? Can't we make it a function and call it in both places?